src/BizHawk.Common/BizHawk.Common.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<TargetFrameworks>netstandard2.0;net8.0</TargetFrameworks>

```

Comment:
Just mentioning that this in only necessary because we have BCL backports which get included in the `netstandard2.0` assemblies. All the other libs can target `netstandard2.0` only and the build succeeds.

Reply:
Relevant, not sure if useful: https://learn.microsoft.com/en-us/dotnet/fundamentals/apicompat/package-validation/overview